src/Symfony/Bundle/WebProfilerBundle/composer.json

```diff
@@ -4,7 +4,7 @@
     "description": "Symfony WebProfilerBundle",
     "keywords": [],
     "homepage": "https://symfony.com",
-    "license": "MIT",
+    "license": "(MIT and CC-BY-3.0)",
```

Comment:
Array of licenses is for disjunctive licenses when users have a choice between licenses.  I believe there is no choice between licenses in this bundle and they are conjunctive licenses.